phone/src/ui/components/Modal.tsx

```diff
@@ -5,7 +5,6 @@ import { Paper } from '@material-ui/core';
 const useStyles = makeStyles(() => ({
```

Comment:
Should be able to just pass an object without using an anon function when the `ThemeObject` does not need to be passed.
```ts
makeStyles({})
```